db/equity_column_mapping

```diff
@@ -0,0 +1,35 @@
+total_acres acres_tota
```

Comment:
ℹ️ When loading the shapefile, I remapped the semi-nonsensical variable names to normal names. Just in case we ever need to do it again, here's the mapping file that the conversion script uses.

Reply:
ℹ️ Here in github, you can't view db/equity_composites.sql, but I put a comment at the top with the command-line options I used for doing the conversion and using this file.

Reply:
Question: Should we add documentation to the README pointing folks to this .sql file and also documented why we chose the composite index data set + the shape files? I can't see the comment here but I would advise that it contains all the necessary URLs and information to re-create the SQL file, either with these data sets or with different data sets (i.e. the UW geography data).

Reply:
Good suggestion. I added a dozen or two lines to the README on why we chose that data, where it came from, and how to update it.